src/getting-started/create-bootable-media/bootable-usb-using-windows.md

```diff
@@ -1 +1,101 @@
-Users creating installation media from Windows can use Rufus. Rufus is an open source startup disk creator capable of writing ISO images to portable storage media.
\ No newline at end of file
+# Create a Bootable Pop!\_OS USB in Windows 10
+
+This section explains the process of creating a bootable Pop!_OS USB in macOS using balena Etcher.
```

Comment:
Ether's website seems to use `balenaEtcher` (with no space) when referring to it using the full name.